node/npm/glide/index.ts

```diff
@@ -175,16 +176,13 @@ function initialize() {
         InfBoundary,
         KeyWeight,
         Boundary,
-        UpdateOptions,
         ProtocolVersion,
         RangeByIndex,
         RangeByScore,
         RangeByLex,
         ReadFrom,
         ServerCredentials,
-        SortClusterOptions,
         SortOptions,
-        SortedSetRange,
```

Comment:
`SortedSetRange` is not exported
https://github.com/valkey-io/valkey-glide/blob/d75a21eafa4ac3b3b593019c5d294c56d8c97cda/node/src/Commands.ts#L1731